assets/js/components/settings/settings-module.js

```diff
@@ -42,7 +42,7 @@ import {
 	activateOrDeactivateModule,
 	getReAuthURL,
 	showErrorNotification,
-	getModulesData,
+	getModulesData, listFormat,
```

Comment:
Nitpick, but let's not mix a single `import` statement to use both one entry per line vs all in the same line. This one should be moved to a new line like the other ones in the same import group are.